.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
This file is auto-created by husk. Seems it is required because husky makes a temp directory name `_`.